etc/Makefile

```diff
@@ -23,6 +23,9 @@ ESSR_CODE_FILES = ESSR/R/*.R ESSR/R/.*.R
 
 all:
 
+.PHONY: dist
+dist: all
```

Comment:
I don't think we need this.

Reply:
You will see a `make dist` in the `rel-staging` target in the parent Makefile.  It calls `make dist` for `lisp`, `etc` and `doc` (thus resolving the pdfs and html).